### ISSUE TITLE
Uploads are marked public as default

### ISSUE THREAD

**Issue by applepipe** [frappe/erpnext#10836]
Hello,

Uploads where marked as private before, but since last week all upload are public unless ou uncheck the box before uploading, the upload is public. Making it private as default could prevent some users to make some sensitive documents public.

**Comment by joolsr** [comment]
Yes, I can confirm this. Email attachments are however stored in /private

**Comment by ddavidebor** [comment]
This issue is still relevant
